sec_edgar_downloader/_utils.py

```diff
@@ -300,6 +307,23 @@ def download_filings(
                         f"'{filing.accession_number}' due to network error: {e}."
                     )
 
+                try:
+                    download_and_save_filing(
```

Comment:
`download_and_save_filing` needs to be modified to actually download the zip folder. In its current state, nothing will be downloaded.

Reply:
Please clarify? I tested this and it does appear to work correctly.

Reply:
Oh my bad, I totally misread this hah! I see what you are doing and this totally makes sense :).

Reply:
Cool. I am going to mark this conversation resolved.